README.md

```diff
@@ -1 +1,7 @@
-# BazaCD
\ No newline at end of file
+# BazaCD
+Desktop application for creating and storing information about music albums in format of text files.
+After choosing 1 option from menu the folder ("Album DB") for storing files, with one text file without records , will be created on partition D of hard drive. 
```

Comment:
Would change it a little:
Initial folder ("Album DB") with one text file without records will be created on hard drive after first data save.
Using console interface user can perform actions as: loading & saving different lists with albums and all CRUD operations.